Sources/StreamChat/Workers/UserListUpdater.swift

```diff
@@ -63,3 +67,30 @@ enum UpdatePolicy {
     /// The resulting user set of the query will replace the existing user set.
     case replace
 }
+
+@available(iOS 13.0.0, *)
+extension UserListUpdater {
```

Comment:
Followed the same approach we have for channel lists where there is an update method and pagination conveniences.